totalopenstation/tests/test_geojson.py

```diff
@@ -18,5 +18,5 @@ def setUp(self):
 
     def test_output(self):
         self.output = OutputFormat(self.data).process()
-        ref_output = '''{"type": "FeatureCollection", "features": [{"geometry": {"type": "Point", "coordinates": [12.8, 76.3, 56.2]}, "type": "Feature", "properties": {"desc": "TEST POINT"}, "id": 1}, {"geometry": {"type": "Point", "coordinates": [19.8, 26.3, 46.2]}, "type": "Feature", "properties": {"desc": "TEST POINT #2"}, "id": 2}]}'''
+        ref_output = '''{"type": "FeatureCollection", "bbox": [12.8, 26.3, 56.2, 19.8], "features": [{"bbox": [12.8, 76.3, 56.2, 12.8, 76.3, 56.2], "geometry": {"type": "Point", "coordinates": [12.8, 76.3, 56.2]}, "type": "Feature", "properties": {"desc": "TEST POINT"}, "id": 1}, {"bbox": [19.8, 26.3, 46.2, 19.8, 26.3, 46.2], "geometry": {"type": "Point", "coordinates": [19.8, 26.3, 46.2]}, "type": "Feature", "properties": {"desc": "TEST POINT #2"}, "id": 2}]}'''
```

Comment:
I don't understand the motivation of this change.

We can add more than one test if you think it's important to test bounding box functionality.

Reply:
If I remember correctly, when I proceded the test with the last pygeoif 0.7, the bbox was added.
See https://github.com/cleder/pygeoif/pull/12

Reply:
Now I see that this same change was already merged in 1bbe97dd6c3e0ec7fc81fa017e679a77bc0a9ad8 👍

Reply:
For this I need to rebase the LandXML branch on master.
Commit is not 1bbe97dd6c3e0ec7fc81fa017e679a77bc0a9ad8 but 28b328ad32a3197e6dd8dcc4b148d3e36f556ab7.